ddl/reorg.go

```diff
@@ -215,6 +338,23 @@ func (d *ddl) getReorgInfo(t *meta.Meta, job *model.Job, tbl table.Table) (*reor
 			return info, errors.Trace(err)
 		}
 
+		reorgMeta := model.NewDDLReorgMeta()
+		reorgMeta.StartHandle = info.Handle
+		// Gets the real table end handle, the new added row after the index being writable,
+		// has no needs to backfill.
+		endHandle, emptyTable, err1 := d.GetTableMaxRowID(ver.Ver, tbl.Meta())
+		if err1 != nil {
+			return info, errors.Trace(err1)
+		}
+
+		if endHandle < reorgMeta.StartHandle || emptyTable {
+			endHandle = reorgMeta.StartHandle
+		}
+
+		reorgMeta.EndHandle = endHandle
+		log.Infof("[ddl] job(%v) get table startHandle:%v, endHandle:%v", job.ID, reorgMeta.StartHandle, reorgMeta.EndHandle)
```

Comment:
s/job(%v)/job /
I think we'd better unify the log format. It's easy to `grep` some information.

Reply:
Agree